app/invocation/workflow_rerun_button.tsx

```diff
@@ -186,6 +210,45 @@ export default class WorkflowRerunButton extends React.Component<WorkflowRerunBu
     );
   }
 
+  // getWorkflowExecution returns the execution for the workflow run (ci_runner).
+  private async getWorkflowExecution(): Promise<execution_stats.Execution> {
+    const executionRequest = new execution_stats.GetExecutionRequest();
+    executionRequest.executionLookup = new execution_stats.ExecutionLookup();
+    executionRequest.executionLookup.invocationId = this.props.model.getInvocationId();
+    const executionResponse = await rpcService.service.getExecution(executionRequest);
+
+    if (executionResponse.execution.length != 1) {
+      throw new Error(`expected 1 workflow execution, got ${executionResponse.execution.length}`);
+    }
+    const workflowExecution = executionResponse!.execution[0];
+    if (!workflowExecution.commandSnippet.startsWith("buildbuddy_ci_runner")) {
```

Comment:
Would probably be better to do this filtering on the Command proto - the command snippet is a stylized representation of the command and isn't super well structured (e.g. arg0 is actually `./buildbuddy_ci_runner` but we currently stylize it to `buildbuddy_ci_runner`, and I could see that changing in the future)

Reply:
Just going to remove this. It's probably unnecessary as an extra check because the execution for the parent workflow is always for the ci_runner (if not, there's some other bigger bug going on)